net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
+PATCHES="yaddns-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	yaddns$secondaryArchSuffix = $portVersion
+	cmd:yaddns$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
```

Comment:
For the test on x86_gcc2 you need to add lib:libcrypto$secondaryArchSuffix here, then all checks pass ...

Reply:
There is something even better to do: instead of adding `lib:libcrypto$secondaryArchSuffix` to `REQUIRES`, we can add `devel:libcrypto$secondaryArchSuffix` to `BUILD_REQUIRES`.